graphql-client/test/Data/GraphQL/Test/Error.hs

```diff
@@ -0,0 +1,37 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE QuasiQuotes #-}
+
+module Data.GraphQL.Test.Error where
+
+import Control.Exception (Exception(..))
+import Data.Aeson.KeyMap (fromList)
+import qualified Data.Aeson.Types as Aeson
+import Test.Tasty (TestTree, testGroup)
+import Test.Tasty.HUnit (testCase, (@?=))
+
+import Data.GraphQL.Error (GraphQLError (..), GraphQLException (..), GraphQLErrorLoc (..))
+
+testErrorDisplay :: TestTree
+testErrorDisplay =
+  testGroup
+    "GraphQLException"
+    [ testCase "displayException" $ do
+        let err =
+              GraphQLException
+                [ GraphQLError
+                    -- TODO: What do actual error messages look like?
+                    { message = "Something went wrong!"
+                    , locations = Just [GraphQLErrorLoc { errorLine = 10, errorCol = 3 }]
+                    , path = Just [ Aeson.String "my_query.graphql" ]
+                    , extensions = Just (Aeson.Object (fromList [("complexity", Aeson.Number 2)]))
+                    }
+                ]
+        displayException err @?=
+          "GraphQL errors:\n"
```

Comment:
Nit: I tend to find unlines makes this more readable?

Reply:
`unlines` adds another `\n` at the end of the output, which we don't want (it leads to blank lines after the output when `displayException` output is `putStrLn`'ed, for example).

I tweaked the style to not use blank lines between individual errors though, let me know what you think.